### PR TITLE
Refs #3139. Added topic profile support.

### PR DESCRIPTION
**Pull request by julianbermudez**
Added topic profile support for XML.

### CODE REVIEW

**Comment by julianbermudez** [comment]
Sorry, I fixed it.

File: include/fastrtps/attributes/PublisherAttributes.h [hunk 1]
```diff
@@ -84,13 +84,13 @@ class PublisherAttributes
          * Set the user defined ID
          * @param id User defined ID to be set
          */
-        inline void setUserDefinedID(uint8_t id){m_userDefinedID = id;	};
+        inline void setUserDefinedID(uint8_t id) { m_userDefinedID = id;	}
```
Comment (MiguelCompany) [inline include/fastrtps/attributes/PublisherAttributes.h]: Pleas remove also the TAB character

File: src/cpp/xmlparser/XMLProfileManager.cpp [hunk 2]
```diff
@@ -81,6 +68,33 @@ XMLP_ret XMLProfileManager::fillSubscriberAttributes(const std::string &profile_
...
+void XMLProfileManager::getDefaultPublisherAttributes(PublisherAttributes& publisher_attributes)
+{
+    publisher_attributes = default_publisher_attributes;
+}
+
+void XMLProfileManager::getDefaultSubscriberAttributes(SubscriberAttributes& subscriber_attributes)
+{
+    subscriber_attributes = default_subscriber_attributes;
+}
+
```
Comment (MiguelCompany) [inline src/cpp/xmlparser/XMLProfileManager.cpp]: I think we should also add a getDefaultTopicAttributes method

File: include/fastrtps/attributes/PublisherAttributes.h (hunk 1 above)
Comment (julianbermudez) [reply include/fastrtps/attributes/PublisherAttributes.h]: Done!

File: src/cpp/xmlparser/XMLProfileManager.cpp (hunk 2 above)
Comment (julianbermudez) [reply src/cpp/xmlparser/XMLProfileManager.cpp]: Done!

**Pull request merged**